LICENSE.bin

```diff
@@ -460,3 +460,8 @@
 
    AOP Alliance
    XZ For Java
+
+   This product bundles various third-party components also under the
```

Comment:
Is this duplicated with 404Line?

Reply:
Yes, but it is repeated many times, please look at it again.

Reply:
It's not repeated for many times.

Reply:
Yeah, it seems already in L404.

Reply:
Let me see

Reply:
Yeah, I seems that this is not an issue. I will close it as it has been fixed.